articles/azure-functions/functions-how-to-github-actions.md

```diff
@@ -41,19 +41,26 @@ In this example, replace the placeholders in the resource with your subscription
 
 ## Download the publishing profile
 
-You can download the publishing profile of your functionapp, by going to the **Overview** page of your app and clicking **Get publish profile**.
+You can download the publishing profile of your function app, by going to the **Overview** page of your app and clicking **Get publish profile**.
 
    ![Download publish profile](media/functions-how-to-github-actions/get-publish-profile.png)
 
 Copy the content of the file.
 
 ## Configure the GitHub secret
 
-1. In [GitHub](https://github.com), browse your repository, select **Settings** > **Secrets** > **Add a new secret**.
+1. In [GitHub](https://github.com), browse to your repository, select **Settings** > **Secrets** > **Add a new secret**.
 
    ![Add Secret](media/functions-how-to-github-actions/add-secret.png)
 
-1. Use `AZURE_CREDENTIALS` for the **Name** and the copied command output for **Value**, if you then select **Add secret**. If you are using publishing profile, use `SCM_CREDENTIALS` for the **Name** and the file content for **Value**.
+1. Add a new secret
```

Comment:
Since there are two ways to specify the credentials, I broke them out to use bullet items